blocks/form/components/accordion/accordion.js

```diff
@@ -1,11 +1,17 @@
-export function handleAccordionNavigation(panel, tab) {
+export function handleAccordionNavigation(panel, tab, forceOpen = false) {
   const accordionTabs = panel?.querySelectorAll(':scope > fieldset');
   accordionTabs.forEach((otherTab) => {
     if (otherTab !== tab) {
       otherTab.classList.add('accordion-collapse');
     }
   });
-  tab.classList.toggle('accordion-collapse');
+  if (forceOpen) {
```

Comment:
write a tertiary operator, easier to read

Reply:
ok

Reply:
Our eslint rules `no-unused-expressions` does not allow that currently. Debatable to have it this way or the other so I'm sticking to this for now.

Reply:
Ok